client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/mongo/MongoReplacementClass.java

```diff
@@ -0,0 +1,30 @@
+package org.evomaster.client.java.instrumentation.mongo;
+
+import com.mongodb.client.ClientSession;
+import com.mongodb.client.FindIterable;
+import com.mongodb.client.MongoCollection;
+import org.bson.conversions.Bson;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.MethodReplacementClass;
+
+public class MongoReplacementClass implements MethodReplacementClass {
+
+    @Override
+    public Class<?> getTargetClass() {
+        return
+                MongoCollection.class;
```

Comment:
unfortunately, we cannot have it like this :(

Reply:
I fixed this one with a Class.forName(...) invokation